lib/organization/team/index.js

```diff
@@ -0,0 +1,153 @@
+import cloneDeep from 'lodash/cloneDeep'
+import {
+  create,
+  fetch,
+  deleteEntity,
+  fetchAll
+} from '../../entity'
+import { TeamUsers } from './teamUsers'
+import { StackRoleMappings } from './stackRoleMapping'
+import error from '../../core/contentstackError'
+
+export function Teams (http, data) {
```

Comment:
Make sure the directory name is same as that of class name. Change this to teams/index.js